connectionpool.go

```diff
@@ -203,7 +205,7 @@ func (p *policyConnPool) addHost(host *HostInfo) {
 	}
 	p.mu.Unlock()
 
-	pool.fill()
+	pool.debouncer.Debounce(pool.fill())
```

Comment:
Could you please move this code to `pool.fill_debounce`:
```suggestion
	pool.fill_debounce()
```